manager/base.py

```diff
@@ -564,3 +565,25 @@ def cyndi_query(query):
     if CYNDI_ENABLED:
         query = query.join(InventoryHosts, JOIN.INNER, on=(SystemPlatform.inventory_id.cast("UUID") == InventoryHosts.id))
     return query
+
+
+def get_advisories_per_cve_from_db(synopsis: str):
+    """Get advisories per CVE from DB"""
+    try:
+        data = (CveMetadata.select(CveMetadata.advisories_list)
+                .where(CveMetadata.cve == synopsis)).dicts()[0]['advisories_list']
+    except IndexError:
+        data = []
+    return data
+
+
+def filter_by_advisory(advisory_name, data):
+    """Filter data by advisory name"""
+    filtered = {}
+    for key, advisories in data.items():
+        for advisory in advisories:
+            pattern = re.compile(r'.*{}.*'.format(advisory_name), re.I)
```

Comment:
please don't use regexp for such simple substring search, regexps are slower than `in` operator

`if keyword.lower() in text.lower()` is simplier

Reply:
@blayson ping ^^